src/renderer/modal/addLocalRepository.vue

```diff
@@ -86,6 +87,8 @@ export default {
 	methods: {
 		fileSectorInput(event) {
 			this.pathToRepository = event.target.files[0].path.split("\\").join("/");
+			var repositoryPathInput = document.getElementById("pathToRepository");
```

Comment:
Can you try doing it with refs?

Reply:
I tried using refs first but that didn't fix the issue. I added a ref="repositoryPath" to the inputText element and then put the following code in the fileSectorInput method:
```
this.$refs.repositoryPath.value = this.pathToRepository;
```
Do you have any suggestions as to why using the refs didn't work?
Thanks.

Reply:
Let me take a look into that.